pages/blog/_posts/vercel-cloudflare-wait-until.mdx

```diff
@@ -0,0 +1,126 @@
+---
+heading: When to use Vercel and Cloudflare's waitUntil utility
+subtitle: What is it, when to use it, and when not to use it
+image: /assets/blog/vercel-cloudflare-wait-until/featured-image.png
+date: 2024-05-16
+author: Dan Farrelly
+---
+
+Recently, [Vercel announced](https://vercel.com/changelog/waituntil-is-now-available-for-vercel-functions) that their `waitUntil` utility method is now available for all serverless functions. Now, this useful feature is available for both Vercel's Node.js and edge functions. A similar utility is also available for [Cloudflare Workers](https://developers.cloudflare.com/workers/runtime-apis/context/#waituntil).
+
+This is a very useful tool, but it's important to understand when to use it and when not to use it. In this post, I'll explain:
```

Comment:
```suggestion
This is a very useful tool so let's dive into what `waitUntil` is and what it does, when to use it, and when it's better not to.
```

Changing it so it's sounds more like you are excited about it, and not telling others to be that.